devito/compiler.py

```diff
@@ -192,7 +192,7 @@ def __init__(self, *args, **kwargs):
         default = '-O3 -g -march=native -fPIC -mno-avx -Wall -std=c99'
         self.cflags = environ.get('CFLAGS', default).split(' ')
         self.ldflags = environ.get('LDFLAGS', '-shared').split(' ')
-
+        self.pragma_ivdep = [Pragma(environ.get('DEVITO_PRAGMA', 'GCC ivdep'))]
```

Comment:
I reiterate this sentiment on the use of `t` vs `time` in the entire codebase :)